src/pages/sangyoungHa/Main/Comment.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class Comment extends Component {
+  render() {
+    let date = new Date();
```

Comment:
```suggestion
    const date = new Date();
```
const로 쓰지 않을 이유가 없을거 같아요!

Reply:
변경했습니다!